tests/test_skvbc.py

```diff
@@ -85,49 +81,50 @@ def test_get_block_data(self):
         trio.run(self._test_get_block_data)
 
     async def _test_get_block_data(self):
-        config = bft.TestConfig(n=4,
-                                f=1,
-                                c=0,
-                                num_clients=10,
-                                key_file_prefix=KEY_FILE_PREFIX,
-                                start_replica_cmd=start_replica_cmd)
-        with bft.BftTestNetwork(config) as bft_network:
-            await bft_network.init()
-            [bft_network.start_replica(i) for i in range(3)]
-            p = self.protocol
-            client = bft_network.random_client()
-            last_block = p.parse_reply(await client.read(p.get_last_block_req()))
-
-            # Perform an unconditional KV put.
-            # Ensure keys aren't identical
-            kv = [(bft_network.keys[0], bft_network.random_value()),
-                  (bft_network.keys[1], bft_network.random_value())]
-
-            reply = await client.write(p.write_req([], kv, 0))
-            reply = p.parse_reply(reply)
-            self.assertTrue(reply.success)
-            self.assertEqual(last_block + 1, reply.last_block_id)
-
-            last_block = reply.last_block_id
-
-            # Get the kvpairs in the last written block
-            data = await client.read(p.get_block_data_req(last_block))
-            kv2 = p.parse_reply(data)
-            self.assertDictEqual(kv2, dict(kv))
-
-            # Write another block with the same keys but (probabilistically)
-            # different data
-            kv3 = [(bft_network.keys[0], bft_network.random_value()),
-                   (bft_network.keys[1], bft_network.random_value())]
-            reply = await client.write(p.write_req([], kv3, 0))
-            reply = p.parse_reply(reply)
-            self.assertTrue(reply.success)
-            self.assertEqual(last_block + 1, reply.last_block_id)
-
-            # Get the kvpairs in the previously written block
-            data = await client.read(p.get_block_data_req(last_block))
-            kv2 = p.parse_reply(data)
-            self.assertDictEqual(kv2, dict(kv))
+        for bft_config in bft.interesting_configs():
+            config = bft.TestConfig(n=bft_config['n'],
+                                    f=bft_config['f'],
+                                    c=bft_config['c'],
+                                    num_clients=bft_config['num_clients'],
+                                    key_file_prefix=KEY_FILE_PREFIX,
+                                    start_replica_cmd=start_replica_cmd)
+            with bft.BftTestNetwork(config) as bft_network:
+                await bft_network.init()
+                [bft_network.start_replica(i) for i in range(config.n)]
```

Comment:
Is this the same as `bft_network.start_all_replicas()` ?

Reply:
Good catch - indeed!